.gitignore

```diff
@@ -1,49 +1,46 @@
 # NOTE: We use Hatch to build wandb, and it respects the *first* .gitignore
 # file it sees. It doesn't consider .gitignore files in subdirectories.
 
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+
+# Distribution / packaging
+dist/
+build/
+*.egg
+
 # Conventional location for Python virtual environments.
 .venv/*
 
+# Configuration files
+.python-version
+.idea/
+# .vscode/
+
 # nox cache
 .nox/
 .nox-wandb/
 
-# Test results.
+# Unit test / coverage reports / Linting
 test-results/
+.pytest_cache/
+.hypothesis/
+
 .coverage
 coverage.xml
 coverage.txt
+cover-results/
 
-# Linting.
 mypy-results/
-
-# Build output in apple_stats.
-apple_stats/.build/
-
-__pycache__/
-htmlcov/
 .mypy_cache/
-.pytest_cache/
-.hypothesis/
-.circle_cache/
+
 .yea_cache/
-build/
-dist/
-*.pyc
-.python-version
-wandb.egg-info
-docs/themes/
-docs/.generated-files.txt
-docs/config.toml
-tests/unit_tests_old/logs/*
-!tests/unit_tests_old/logs/cleanup.sh
-pip-wheel-metadata
-.vscode/.ropeproject
-xcuserdata/
-cover-results/
-wandb/proto_check/
-*.log
-.idea/
 .yea-requirements.txt
```

Comment:
afuera?

Reply:
yes, i have a pr that removed left over yea stuff

Reply:
#8384